src/main/scala/com/codacy/client/bitbucket/v2/service/HookServices.scala

```diff
@@ -32,18 +34,22 @@ class HookServices(client: BitbucketClient) {
       hookUrl: String,
       events: Set[String]
   ): RequestResponse[v2.Webhook] = {
-    val servicesUrl = getServicesUrl(author, repo)
+    val servicesUrl = generateHooksUrl(author, repo)
+    val encodedUuid = URLEncoder.encode(uuid, "UTF-8")
```

Comment:
Codacy found an issue: [The string literal 'UTF-8' appears 4 times in the file.](https://app.codacy.com/gh/codacy/bitbucket-scala-client/pullRequest?prid=5907044)